tests/src/test/scala/cats/tests/FoldableTests.scala

```diff
@@ -113,6 +114,25 @@ class FoldableTestsAdditional extends CatsSuite {
     val large = Stream((1 to 10000): _*)
     assert(contains(large, 10000).value)
   }
+
+  test(".foldLeftM") {
+    // you can see .foldLeftM traversing the entire structure by
+    // changing the constant argument to .take and observing the time
+    // this test takes.
```

Comment:
Right, it became obsolete earlier, though. [See my comment.](https://github.com/TomasMikula/cats/commit/3eca17fd1f05d861599ce6a697d96ae8a6c5b762#commitcomment-17900340) The whole test case is redundant in favor of the one below.